app/models/concerns/interval.rb

```diff
@@ -20,6 +20,23 @@ module Interval
     scope :overlapping_with, ->(period) { where(*overlapping_with_range(period.started_on, period.finished_on)) }
     scope :containing_period, ->(period) { where(*containing_range(period.started_on, period.finished_on)) }
     scope :ongoing_on, ->(date) { where(*date_in_range(date)) }
+    scope :ongoing_on_including, ->(date) { started_on_or_before(date).and(finished_on_or_after(date)) }
```

Comment:
Do we need this after the catch up with Pete? or can we just change the behaviour of `ongoing_on` to be inclusive?